mtk_logger.cc

```diff
@@ -227,6 +227,7 @@ static char* OPT_erase;  /* erase ? command option */
 static char* OPT_erase_only;  /* erase_only ? command option */
 static char* OPT_log_enable;  /* enable ? command option */
 static char* csv_file; /* csv ? command option */
+static char* OPT_block_size; /* block_size ? command option */
```

Comment:
Does the block size have to be powers of 2 or multiples of 1024 or have any other limits or can it be free form?